### DISCUSSION TITLE
Daily 25/10

### DISCUSSION THREAD

**Discussion by rpgrca** [rpgrca/proyecto-ispc-fullstack#229] [General]
Ayer hice varias modificaciones al proyecto en la parte de Python, cambié los UUID que normalmente se usan por ids de base de datos (lo que es una falla de seguridad pero por el nivel en el que estamos es lo que dijimos que íbamos a hacer de acuerdo al diagrama de clases). También terminé con todo lo que tiene que ver con la subasta excepto lo relacionado con las pujas. Hoy voy a ver qué no tomaron de Python para continuar haciendo la parte de back-end.

**Comment by NataliaAlvarezIspc** [comment]
Ayer estuve estudiando Python y Full. Comencé con la tarea de Python del repo pensando en terminarla hoy.
Hoy estuve con inconvenientes personales que me llevaron todo el día, así que no pude realizar ninguna tarea. Mas tarde intentaré corregir  lo de queries, sino lo haré mañana.

**Comment by AhumadaCeleste** [comment]
Ayer rendí, me asigne tareas de sprint 2 y comencé a estudiar java y a a preparar ara el examen de fullstack
